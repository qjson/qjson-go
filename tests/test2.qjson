x: (2+3) * 2
d 1: 2m 2
d 2: 1h 1m 1s
s: "abcd"
v: [23, 2+1h, 'alt\b </tag>', "alt\f </tag>", alt; </tag>]