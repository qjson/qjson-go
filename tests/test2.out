{"x":10,"d 1":122,"d 2":3661,"s":"abcd","v":[23,3602,"alt\b <\/tag>","alt\f <\/tag>","alt; <\/tag>"]}
